OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -80,13 +80,17 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
         adapter = object : MyCoursesListAdapter(environment) {
             override fun onItemClicked(model: EnrolledCoursesResponse) {
                 activity?.let { activity ->
-                    environment.router.showCourseDashboardTabs(activity, model, false)
+                    environment.router.showCourseDashboardTabs(activity, model)
                 }
             }
 
             override fun onAnnouncementClicked(model: EnrolledCoursesResponse) {
                 activity?.let { activity ->
-                    environment.router.showCourseDashboardTabs(activity, model, true)
```

Comment:
please confirm `Screen.COURSE_ANNOUNCEMENT`, and parameter `announcements`, performs the same functionality.

Reply:
Conceptually, Yes. They both perform the same functionality. 

But the `announcements` parameter was never utilized in the code base. It was passed to `CourseTabsDashboardActivity` but never retrieved or used. 

So we created a new method to pass the screen name instead of a simple boolean value but this time retrieved and utilized.